### ISSUE TITLE
'Error: ENOENT, no such file or directory' on new project 'first run'

### ISSUE THREAD

**Issue by zenlambda** [reapp/reapp#25]
Reproducer / use case

```
» reapp new myreapp
» cd myreapp
» reapp run
```

Results in error:

```
Error! Make sure you have run npm install in reapp-pack
Error! Make sure you have run npm install in reapp-pack

fs.js:438
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^
Error: ENOENT, no such file or directory 'path/to/reapp/myreapp/assets/layout-web.html'
    at Object.fs.openSync (fs.js:438:18)
    at Object.fs.readFileSync (fs.js:289:15)
    at makeLayout (/usr/local/lib/node_modules/reapp/node_modules/reapp-pack/lib/makeLayout.js:12:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/reapp/bin/reapp-run:49:14)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)

```

Basically the reapp cli is looking for layout-web.html and not layout.html by default. Guess this could be fixed by altering reapp-starter accordingly.

**Comment by zenlambda** [comment]
For users the work-around seems to be as follows:

```
 mv assets/layout.html mv assets/layout-web.html 
```

You may also run into #26 after applying the workaround.

**Comment by natew** [comment]
@zenlambda what a mess I got myself into :)

I started work on a new branch `new-config` that will become 0.10, and must have pushed out a patch thinking I was on master at some point.

I really appreciate this request, but actually what I'm going for in 0.10 has moved away from the `layout-` setup and instead will use subdirectories.

It's not quite ready for release yet, so what I've done for now is reverted to the syntax before. It should work with `layout.html` and `layout.build.html`. I do appreciate the help though, let me know if this is still broken in any way for you.
